src/exo/API_scheduling.py

```diff
@@ -1372,8 +1372,6 @@ def divide_dim(proc, alloc_cursor, dim_idx, quotient):
         `x : R[n, 3, 4, m]`
         `x[i, j / 4, j % 4, k] = ...`
     """
-    if quotient == 1:
```

Comment:
Can you change the docstring above "This limited implementation of `divide_dim` requires that the dimension being divided is constant itself." ?

Reply:
Done!